RELEASE_NOTES.md

```diff
@@ -1,5 +1,15 @@
-# Chef Client Release Notes 12.6.0:
+# Chef Client Release Notes 12.7:
+
+## Updates to versioning strategy
+
+TODO
+https://github.com/chef/chef-rfc/blob/master/rfc047-release-process.md
 
+## Net-ssh updates
+
+We updated the version of [net-ssh](https://github.com/net-ssh/net-ssh) in Chef from version 2.9 to 3.0 to take in an upstream bug fix.  The biggest change here is that they dropped support for Ruby 1.9 (which Chef already dropped support for).  Because this is such a low level dependency we found that many other projects had to be updated in lock-step (like Test Kitchen and Berkshelf) for the ChefDK packaging to succeed without dependency conflicts.
+
+# Chef Client Release Notes 12.6.0:
```

Comment:
I removed the 12.6 notes - thanks!

And I absolutely agree that the chocolatey_package and zypper multipackage changes should get a section.  I'm calling on all contributors to add their own RELEASE_NOTES since there isn't a way I could know everything that needs to go in here.

@lamont-granquist I see you made the zypper changes - would you mind adding a commit to this branch or pinging me an explanation in slack of what the change is?